### ISSUE TITLE
About the finetuning

### ISSUE THREAD

**Issue by leiyu-celeb** [wentaozhu/AnatomyNet-for-anatomical-segmentation#25]
hello, sir, a nice work about your hybrid loss func!
I just want to figure out  whether to use the hybrid loss function(dice + focal loss) in pre-training(use  baselineSERes18Conc.py to initialize weights) and also in fine-tuning stage(load pretrained model, use AnatomyNet.py)，I am looking forward to your reply, thanks a lot.
